src/resources/views/crud/buttons/quick.blade.php

```diff
@@ -37,3 +55,44 @@
         {{ $label }}
     </{{ $wrapper['element'] }}>
 @endif
+
+
+@if($ajax_enabled)
+{{-- Button Javascript --}}
+{{-- Pushed to the end of the page, after jQuery is loaded --}}
+@push('after_scripts') @if (request()->ajax()) @endpush @endif
+@bassetBlock('backpack/crud/buttons/quick-button.js')
+<script>
+	if (typeof sendQuickButtonAjaxRequest != 'function') {
+	  $("[data-button-type=quick]").unbind('click');
+
+	  function sendQuickButtonAjaxRequest(button) {
+		// e.preventDefault();
+		var route = $(button).attr('data-route');
+				$.ajax({
+			      url: route,
+			      type: $(button).attr('data-method'),
+			      success: function(result) {
+			        new Noty({
+		            	type: "success",
+						text: result.message?result.message:'<strong>'+$(button).attr('data-success-title')+'</strong><br>'+$(button).attr('data-success-message'),
```

Comment:
Again, it's a preference. One-liner is good. It's not a complicated-looking ternary operation.

I see the same practice in other buttons.

Reply:
It still one liner, but at the moment there is so much to digest in that line that is very difficult to read.

- it's already a ternary operation, so not the most "expressive" one. 
- they keys are nested, I mean `result.message` instead of only `message`. 
- adding html to the string
- adding $() jquery wrappers and selectors

>I see the same practice in other buttons.

We are always learning. Don't let the errors from the past prevent you from doing better in future. 

I would lean to go even further to make it easier to read (those + plus signs still pollute the vision), using template literals:
```javascript
`<strong>${buttonSuccessTitle}</strong><br/>${buttonSuccessMessage}`;
```

I don't think in this case it's a "preference".
` result.message?result.message:'<strong>'+$(button).attr('data-success-title')+'</strong><br>'+$(button).attr('data-success-message'),` is not an easy to read string anywhere in the world.